libs/vkd3d/command.c

```diff
@@ -17782,6 +17816,18 @@ static void d3d12_command_queue_execute(struct d3d12_command_queue *command_queu
         num_submits += 2;
     }
 
+    if (command_queue->device->vk_info.NV_low_latency2 &&
```

Comment:
There are other places where we submit work, such as memory clears. Is there a hard requirement of LL2 that every vkQueueSubmit `must:` be assigned to a present ID? What happens for submissions that aren't tied to a present ID? Do they just not participate in the algorithm? That would be for the best in this case.

Reply:
There is not a requirement to provide a presentID with every call to vkQueueSubmit. You are correct when you say submits that do not contain a present id don't participate in the algorithm. I am guessing the clears don't take up a large amount of GPU time or significantly delay the GPU from doing application work so it should be fine.